test/aqua.jl

```diff
@@ -0,0 +1,36 @@
+using Test
+using TaylorIntegration
+using Aqua
+
+@testset "Aqua tests (performance)" begin
+    # This tests that we don't accidentally run into
+    # https://github.com/JuliaLang/julia/issues/29393
+    # Aqua.test_unbound_args(TaylorIntegration)
+    ua = Aqua.detect_unbound_args_recursively(TaylorIntegration)
+    @test length(ua) == 0
+
+    # See: https://github.com/SciML/OrdinaryDiffEq.jl/issues/1750
+    # Test that we're not introducing method ambiguities across deps
+    ambs = Aqua.detect_ambiguities(TaylorIntegration; recursive = true)
+    pkg_match(pkgname, pkdir::Nothing) = false
+    pkg_match(pkgname, pkdir::AbstractString) = occursin(pkgname, pkdir)
+    filter!(x -> pkg_match("TaylorIntegration", pkgdir(last(x).module)), ambs)
+    for method_ambiguity in ambs
+        @show method_ambiguity
+    end
+    if VERSION < v"1.10.0-DEV"
+        @test length(ambs) == 0
+    end
+end
+
+@testset "Aqua tests (additional)" begin
+    Aqua.test_undefined_exports(TaylorIntegration)
+    Aqua.test_deps_compat(TaylorIntegration)
+    Aqua.test_stale_deps(TaylorIntegration;
+        ignore=[:DiffEqBase, :RecursiveArrayTools, :Requires, :StaticArrays])
+    Aqua.test_piracies(TaylorIntegration)
+    Aqua.test_unbound_args(TaylorIntegration)
+    Aqua.test_project_extras(TaylorIntegration)
+    Aqua.test_persistent_tasks(TaylorIntegration)
+end
```

Comment:
Took a quick look to the Aqua docs, seems like there's a do-it-all function here, but I this is exactly equivalent to call the tests one by one as is it done now?
```suggestion
@testset "Aqua tests (additional)" begin
    Aqua.test_all(
    TaylorIntegration;
    stale_deps=(ignore=[:DiffEqBase, :RecursiveArrayTools, :Requires, :StaticArrays],),
    )
end
```

Reply:
Committed your suggestion; thanks!